requirements/requirements.in

```diff
@@ -1,5 +1,7 @@
 #-c constraints.in # <-- contains known limitations
 
 staticjinja >= 5.0.0  # generates html from templates
+jinja2 >= 3.1.4  # need more recent version than staticjinja, 
```

Comment:
Actually staticjinja does not need this specific version. 3.1.3 was just picked by dependency resolution as the latest jinja2 release that was available when requirements/requirements.txt was built.

So removing this line again and rebuilding requirements/requirements.txt will also give you the fixed version.

(Keeping this one here is also fine, but I would remove the reference to staticjinja, since that package has nothing to do with the jinja2 version chosen. Better reference the CVE here.)